src/rpi-cpu2mqtt.py

```diff
@@ -289,7 +289,8 @@ def config_json(what_config):
         data["state_class"] = "measurement"
     elif what_config == "git_update":
         data["icon"] = "mdi:git"
-        data["name"] = "Update Available"
+        data["name"] = "Script version"
```

Comment:
What about just using the name?
```suggestion
        data["name"] = "rpi-mqtt-monitor"
```